tools/fuchsia/gen_package.py

```diff
@@ -98,10 +100,18 @@ def main():
   # Use check_output so if anything goes wrong we get the output.
   try:
 
+    build_command = ['build', '--output-package-manifest', args.manifest_json_file]
+
+    if args.api_level is not None:
+      build_command = ['--api-level', args.api_level] + build_command
+
+    archive_command = ['archive', '--output='+ os.path.join(os.path.dirname(output_dir), args.far_name + "-0")]
```

Comment:
Can you file a bug to get rid of the -0?  It complicates people's workflows because the fars aren't named that everywhere, only in our repo.  It's also useless since it's just an arbitrary thing we hardcore here